system-uptime-tracking/xml/art_system-uptime-tracking.xml

```diff
@@ -0,0 +1,375 @@
+    %entities;
+  <!ENTITY productnamex "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productname;</phrase><phrase xmlns='http://docbook.org/ns/docbook' os='article'>&sls;</phrase>">
+   <!ENTITY productnumberx "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productnumber;</phrase>">
+]>
+<article xml:id="article-system-uptime-tracking"
+         xmlns="http://docbook.org/ns/docbook"
+         version="5.0"
+         xml:lang="en"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>System Uptime Tracking</title>
+  <titleabbrev>System uptime tracking in &scc; and &rmt;</titleabbrev>
+  <info><productname>&productnamex;</productname>
+    <productnumber>&productnumberx;</productnumber><date>
+<?dbtimestamp format="B d, Y" ?></date>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker>
+        <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+        <dm:product>Documentation</dm:product>
+        <dm:component>SLE Family</dm:component>
+        <dm:assignee>tbazant@suse.com</dm:assignee>
+      </dm:bugtracker>
+      <dm:editurl>https://github.com/SUSE/doc-unversioned/edit/main/system-uptime-tracking/xml/</dm:editurl>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+    <abstract>
+      <para>
+        The system uptime tracking feature monitors when a client system is up.
+        It works on an hourly basis&mdash;the system is considered up if it was
+        running at any part of the given hour. The information is stored in
+        &scc; (SCC) and Repository Mirroring Tool (RMT), and serves as a source
+        for analytic purposes.
+      </para>
+    </abstract>
+  </info>
+  <sect1 xml:id="uptime-requirements">
+    <title>Requirements</title>
+
+    <itemizedlist>
+      <listitem>
+        <para>
+          The version of the <package>suseconnect-ng</package> package has to be
+          1.10 or newer. Verify it by running <command>suseconnect
+          --version</command>.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          If the client system is behind RMT, the version of the
+          <package>rmt-server</package> package has to be 2.17 or newer. Verify
+          it by running <command>rpm -q rmt-server</command>.
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+  <sect1 xml:id="uptime-enable">
+    <title>Enabling the uptime tracking</title>
+
+    <para>
+      The uptime tracking is disabled by default. To enable it on the client
+      system, start its &systemd; timer and adjust the &suseconnect;
+      configuration.
+    </para>
+
+    <procedure>
+      <title>Enabling the &systemd; timer</title>
+      <step>
+        <para>
+          Install the <package>suse-uptime-tracker</package> package.
+        </para>
+<screen>&prompt.sudo;zypper install suse-uptime-tracker</screen>
```

Comment:
we may need to specify where to download this from until MSC-788 is complete, as the suse-uptime-tracker package is not yet part of the regular SLE channels